lib/Events/RemoveParticipantEvent.php

```diff
@@ -31,14 +32,26 @@ class RemoveParticipantEvent extends ParticipantEvent {
 	/** @var string */
 	protected $reason;
 
+	/** @var Session[] */
+	protected $sessions;
+
 	public function __construct(Room $room,
 								Participant $participant,
-								string $reason) {
+								string $reason,
+								array $sessions = []) {
```

Comment:
Since i moved the listener to the before event this modification might not be neededd